packages/mongodb-rag-ingest/src/sources/snooty/SnootyDataSource.ts

```diff
@@ -189,9 +205,11 @@ export const makeSnootyDataSource = ({
             case "asset":
               // Nothing to do with assets (images...) for now
               return;
-            case "metadata":
-              // Nothing to do with metadata document for now
+            case "metadata": {
+              const { data } = entry as SnootyMetadataEntry;
+              siteTitle = handleMetadata(data).title;
```

Comment:
Do we need a `handleMetadata()` function? It seems like an identity function that justs return a copy of what's passed in, i.e. `(metadata: SnootyMetadata) => SnootyMetadata`.

In this case, couldn't we assign directly off of the `data` object?

```suggestion
              siteTitle = data.title;
```

Reply:
Agreed, I made the change 👍